### PR TITLE
Board of directors meeting 2021-12-14

### LINKED ISSUE #697: Provisions Prohibiting Support of Terrorist Activities
- To add to donation page:

G. Provisions Prohibiting Support of Terrorist Activities
1. Institution will not use any portion of any grant from the Fund to engage in, support or promote violence, terrorist activity or related training of any kind, whether directly through its own activities and programs or indirectly through its support of, or cooperation with, other persons and organizations engaged in such activities, and will take reasonable precautions and institute reasonable procedures to prevent any portion of the grant from being so used.
2. Institution certifies that: (a) we do not support violence or terrorist activities; (b) none of our trustees, directors, officers or key staff members support violence or terrorist activities and we do not employ any individuals known to us to support such activities; (c) none of our affiliates or subsidiaries support violence or terrorist activities, we do not engage in transactions of any kind with vendors, contracting organizations or any other entities known to us to support violence or terrorist activities, and we do not otherwise disburse any of our funds, as grants or in any other form, to individuals or entities known to us to support such activities; (d) we have reviewed the lists of individuals and organizations identified by the U.S. government, the United Nations and the European Union as associated with terrorism and we do not appear on those lists; none of our trustees, directors, officers, key staff members or other employees appear on those lists; and none of our affiliates, subsidiaries, vendors, other contracting organizations, grantees or other entities with which we engage in transactions appear on those lists.

### PR DESCRIPTION
**Pull request by bvhme**
Included is

* the action and decision list of the board meeting
* agreed upon guidance for requesting professional development

We've tried to close some outstanding issues
(thanks @clausmullie) by affirming some policy
and thus fixes #710 and fixes #697

-----
[View rendered activities/staff-information/index.md](https://github.com/publiccodenet/about/blob/2021-12-14-board-of-directors/activities/staff-information/index.md)
[View rendered activities/staff-information/request-professional-development.md](https://github.com/publiccodenet/about/blob/2021-12-14-board-of-directors/activities/staff-information/request-professional-development.md)
[View rendered organization/board-of-directors-meetings/2021-12-14-minutes.md](https://github.com/publiccodenet/about/blob/2021-12-14-board-of-directors/organization/board-of-directors-meetings/2021-12-14-minutes.md)
[View rendered organization/board-of-directors-meetings/index.md](https://github.com/publiccodenet/about/blob/2021-12-14-board-of-directors/organization/board-of-directors-meetings/index.md)

### CODE REVIEW

**Review by ericherman — APPROVED** [review]

File: activities/staff-information/request-professional-development.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+type: Guide
+---
+
+# Professional Development
+
+Staff at the Foundation for Public Code is responsible for their professional development and growth.
+This means that staff needs to invest both in time and effort, such as courses, as a part of their job.
+This guide explains what you need to do in order to engage in learning opportunities.
+
+Professional development is fully paid for by the organization, both in time and learning costs,
```
Comment (Ainali) [inline activities/staff-information/request-professional-development.md:11]: Is the line break in the end of this line intentional?

Comment (bvhme) [reply activities/staff-information/request-professional-development.md:11]: Yes.

In order to keep tracking changes easy I'm trying to keep sentence fragments on separate lines so swaps and shifts can be better tracked through Git.

Comment (Ainali) [reply activities/staff-information/request-professional-development.md:11]: Ok! But then you have missed quite a few sentence fragments. Should I mark them?

Comment (bvhme) [reply activities/staff-information/request-professional-development.md:11]: I've looked through all of them again and think this is a good distinction there now.
I think the commas on the same line don't actually fragment the sentences here.

In the future, this guidance could be rewritten to be clearer and less spoken language to remove this confusion.
I suggest leaving it for now, however suggestions are always welcome.

Comment (Ainali) [reply activities/staff-information/request-professional-development.md:11]: Okay, then your linguistic skills are way beyond mine and I hope I will not need to keep to the same standard in my pull requests :)

**Review by nebster — APPROVED** [review]
official approval!

**Pull request merged**